### DISCUSSION TITLE
Synology Docker Install

### DISCUSSION THREAD

**Discussion by eaglefox82** [sct/overseerr#2047] [Help] [answered]
I would love to help and test this application.

However I am new to docker and have never installed a docker image from outside the offical repos.

Can someone please help with a guide of installing this docker image within Docker on a synology NAS.

Thanks.

**Comment by danshilm** [comment, marked as answer]
I'm not familiar with Synologys, but I found this knowledge base article that should cover all the basic things needed to set Overseerr up in a Docker container: [link](https://kb.synology.com/en-us/DSM/help/Docker/docker_container?version=6).

Few things:
- Choosing an image from the Registry: all the details below are for the `sctx/overseerr` image, some things will be different if you choose another image.
- Port Settings: the "Container Port" needs to be `5055`, the "Local Port" can be whatever you choose (just make sure it doesn't conflict with other container ports on your Synology - it can be `5055` too)
- Volumes: you need one volume with the "Mount Path" being `/app/config`. For the "File/Folder", set it to the path to a folder you want to keep the Overseerr configuration data in.
- Environment: you just need one variable set, named `TZ`. The value for it will depend on where you live, refer to this [list of tz database names](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) to find yours.

**Comment by mrfabe** [comment]
I just installed the linuxserver/overseerr image on my Synology and it went very smooth.  The helpful few things posted by @danshilm are the basic things you need to do to install on docker.  Connecting between Plex, Sonaar, and Radarr, everything appears to work as intended.  Just follow the instructions when prompted during the first run.  There were minor interface glitches occurred but nothing that would cripple the use of the program.  I love the work the development team has done on this.
